src/index.html

```diff
@@ -11,7 +11,211 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
-    <script type="text/javascript" src="./main.js"></script>
+    <form method="post" action="https://mate-academy-form-lesson.herokuapp.com/create-application">
+      <fieldset class="inputs-block">
+        <legend>
+          Personal information
+        </legend>
+        <label
+          class="input-row"
+        >
+        Surname:
+          <input 
+            name="surname"
+            autocomplete="off"
+            required=""
+          >
+        </label>
+        <label 
+          class="input-row"
+        >
```

Comment:
де один, максимум 2 атрибути - можна не переносити 
```suggestion
        <label class="input-row">
```